tools/configure

```diff
@@ -2263,6 +2263,8 @@ $as_echo_n "checking OMR_BUILD_ARCH... " >&6; }
 case $build_cpu in #(
   *arm*) :
     OMR_BUILD_ARCH=arm ;; #(
+  aarch64) :
+    OMR_BUILD_ARCH=aarch64 ;; #(
```

Comment:
This is a generated file, and I don't think you updated the source `tools/configure.ac`.

Reply:
Nevermind, this looks good! Jumped the gun a little :)